staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5913,7 +5913,8 @@ type RangeAllocation struct {
 }
 
 const (
-	// "default-scheduler" is the name of default scheduler.
+	ObjectNameField = "metadata.name"
```

Comment:
needs an export comment if added here.
instead of moving one of the object fields constants to here it makes sense to move all of them to fragment less.

yet, unclear whether this is the right location.
given "metadata.*" as a field selector path is unlikely to change, these field constants field a bit redundant and makes more sense to me to define them in locations that need them.

Reply:
done